src/os/ObjectStore.h

```diff
@@ -1523,6 +1523,10 @@ class ObjectStore {
   virtual bool wants_journal() = 0;  //< prefers a journal
   virtual bool allows_journal() = 0; //< allows a journal
 
+  virtual PerfCounters *get_perf_counter() {
```

Comment:
@ifed01 , I will extent this interface for filestore and kstore and split the checkin